server/models/video-blacklist.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
+
+const modelUtils = require('./utils')
+
+// ---------------------------------------------------------------------------
+
+module.exports = function (sequelize, DataTypes) {
+  const BlacklistedVideo = sequelize.define('BlacklistedVideo',
+    {
+      remoteId: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        primaryKey: true,
+        validate: {
+          isUUID: 4
+        }
+      },
+      localId: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        allowNull: false,
+        validate: {
+          isUUID: 4
+        }
+      }
+    },
+    {
+      indexes: [
+        {
+          fields: [ 'remoteId' ]
+        },
+        {
+          fields: [ 'localId' ],
+          unique: true
+        }
+      ],
+      classMethods: {
+        associate,
+
+        countTotal,
+        list,
+        listForApi,
+        loadById,
+        loadByPod
+      },
+      instanceMethods: {
+        toFormatedJSON
+      },
+      hooks: {}
+    }
+  )
+
+  return BlacklistedVideo
+}
+
+// ------------------------------ METHODS ------------------------------
+
+function toFormatedJSON () {
+  return {
+    remoteId: this.remoteId,
+    localId: this.localId,
+    remotePodId: this.remotePodId,
+    createdAt: this.createdAt
+  }
+}
+
+// ------------------------------ STATICS ------------------------------
+
+function associate (models) {
+  this.belongsTo(models.Pod, {
```

Comment:
The `BlacklistedVideos` table just belongs to the `Videos` table (to create the `videoId` foreign key). So:

    this.belongsTo(models.Video, {
      foreignKey: 'videoId',
      onDelete: 'cascade'
    })